### PR TITLE
Dependency version freeze; Pipenv lock, legacy pip requirements.

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -27,10 +27,10 @@ maya = "*"
 #
 # Third-Party Ethereum
 #
-eth-utils="==1.2.2"
+eth-utils="==1.2.1"
```
Comment (cygnusv) [inline Pipfile]: Why is this? AFAIK, v1.2.2 was working.

Comment (KPrasch) [reply Pipfile]: It causes an unresolvable sub-dependency condition with `eth-utils`

File: Pipfile [hunk 2]
```diff
@@ -10,7 +10,7 @@ python_version = "3"
 #
 # NuCypher
 #
-umbral = "==0.1.1a3"
+umbral = "*"
```
Comment (cygnusv) [inline Pipfile]: We need the current release of Umbral, since the next one will break stuff.

```suggestion
umbral = "==0.1.1a3"
```

Comment (EvaLovlac3) [inline Pipfile]: Is this a version wild card?

Comment (KPrasch) [reply Pipfile]: Just fixed in `92dc863`

Comment (KPrasch) [reply Pipfile]: Good Eye

File: setup.py [hunk 3]
```diff
@@ -67,39 +69,9 @@ def run(self):
...
-    'py-solc',
-    'web3',
-
-    # Third Party (Configuration + CLI)
-    'appdirs',
-    'click>=7.0',
-    'colorama',
-    'sentry-sdk==0.5.2',
-    'boto3',
+with open(os.path.join(BASE_DIR, "requirements.txt")) as f:
```
Comment (tuxxy) [inline setup.py]: Is this safe?

Feels kinda odd, are any other projects doing this?

Comment (KPrasch) [reply setup.py]: Yes - many.  What is the concern?

Comment (tuxxy) [reply setup.py]: Just the idea that a modified requirements.txt can cause it to install the depends. Just curious.

**Pull request merged**